site/source/user-manual/backups/backup-setup/index.rst

```diff
@@ -0,0 +1,18 @@
+.. _backup-setup:
+
+=============
+Setup Backups
+=============
+
+.. tip:: Keeping a strong password and making regular backups are the humble responsibilities of digital independence, remain ever vigilant!
+
+Click your platform for detailed instructions on how to create a LAN Shared Folder for creating backups (recommended).  Otherwise, you may backup to a physical drive, attached in addition to your main Embassy SSD.
+
+.. toctree::
+  :maxdepth: 1
+
+  Linux LAN Shared Folder <backup-linux>
+  Mac LAN Shared Folder <backup-mac>
+  Windows  LAN Shared Folder <backup-windows>
+  Synology  LAN Shared Folder <backup-synology>
+  Secondary Embassy Drive <backup-physical>
```

Comment:
This doesn't really belong here, since this is about setting up LAN Shared Folders. Under this structure, physical backups seem like something else entirely.